src/Security/Voter/Committee/EditDeleteCommitteeFeedItemVoter.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace AppBundle\Security\Voter\Committee;
+
+use AppBundle\Committee\CommitteeFeedItemPermissions;
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\CommitteeFeedItem;
+use AppBundle\Security\Voter\AbstractAdherentVoter;
+
+class EditDeleteCommitteeFeedItemVoter extends AbstractAdherentVoter
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function supports($attribute, $subject)
+    {
+        return (CommitteeFeedItemPermissions::EDIT === $attribute
+                || CommitteeFeedItemPermissions::DELETE === $attribute)
+            && $subject instanceof CommitteeFeedItem;
+    }
+
+    /**
+     * @param CommitteeFeedItem $subject
+     */
+    protected function doVoteOnAttribute(string $attribute, Adherent $adherent, $subject): bool
+    {
+        return $subject->getAuthor() !== $adherent || $adherent->isSupervisorOf($subject->getCommittee());
```

Comment:
Please use `$subject->getAuthor()->equals($adherent)` (or the other way around).